README.md

```diff
@@ -12,45 +12,79 @@ NetBox Operator extends the Kubernetes API by allowing users to manage NetBox re
+- Deploy the NetBox Operator on the local kind cluster: `make deploy-kind`
 
-### Using a kind cluster
+### Running the NetBox Operator on your machine and NetBox on a local kind cluster
 
-Follow the instructions to bring up a locally running kind cluster, with NetBox and a NetBox operator running within it:
-- execute `make create-kind`
-- set the following environment varialbes:
+- Create kind cluster with a NetBox deployment: `make create-kind`
+- Port-forward NetBox: `kubectl port-forward deploy/netbox 8080:8080 -n default`
+- Open <http://localhost:8080> in your favorite browser and log in with the username `admin` and password `admin`, you will be able to access the local NetBox instance running in the kind cluster.
+- Open a new terminal window and export the following environment variables:
+
+    ```bash
+    export NETBOX_HOST="localhost:8080"
+    export AUTH_TOKEN="0123456789abcdef0123456789abcdef01234567"
+    export POD_NAMESPACE="default"
+    export HTTPS_ENABLE="false"
+    export NETBOX_RESTORATION_HASH_FIELD_NAME="netboxOperatorRestorationHash"
     ```
-    NETBOX_HOST="localhost:8080"
-    AUTH_TOKEN="0123456789abcdef0123456789abcdef01234567"
-    POD_NAMESPACE="default"
-    HTTPS_ENABLE="false"
-    NETBOX_RESTORATION_HASH_FIELD_NAME="netboxOperatorRestorationHash"
+
+- Run the NetBox Operator locally (in a new terminal window) `make install && make run`
+
+### Running the NetBox Operator on your machine using an existing NetBox and Kubernetes cluster
+
+Note: This requires a running NetBox instance that you can use (e.g. <https://demo.netbox.dev>) as well as a kubernetes cluster (can be as simple as `kind create cluster`)
+
+- Prepare NetBox (based on the demo NetBox instance):
+  - Open <https://demo.netbox.dev/plugins/demo/login/> and create any user
+  - Open <https://demo.netbox.dev/user/api-tokens/> and create a token "0123456789abcdef0123456789abcdef01234567" with default settings
+  - Open <https://demo.netbox.dev/extras/custom-fields/add/> and create a custom field called "netboxOperatorRestorationHash" for Object types "IPAM > IP Address" and "IPAM > Prefix"
+- Open a new terminal window and export the following environment variables (adjust the API :
+
+    ```bash
+    export NETBOX_HOST="demo.netbox.dev"
+    export AUTH_TOKEN="0123456789abcdef0123456789abcdef01234567"
+    export POD_NAMESPACE="default"
+    export HTTPS_ENABLE="true"
+    export NETBOX_RESTORATION_HASH_FIELD_NAME="netboxOperatorRestorationHash"
     ```
-- execute `make deploy-kind`
-- in a separate terminal, run `kubectl port-forward deploy/netbox 8080:8080 -n default`
-- go to your favorite browser and type in `localhost:8080`, with the username `admin` and password `admin`, you will be able to access the local NetBox instance running in the kind cluster
 
-### Using an existing NetBox instance and Kubernetes cluster
+- Run the NetBox Operator locally (in a new terminal window) `make install && make run`
```

Comment:
This should be in the same terminal as the export statements are run else they'll not be picked up